src/components/DetailedProduct/index.js

```diff
@@ -0,0 +1,54 @@
+import { 
+  StyledArticle,
+  StyledImage, 
+  StyledUserIcon,
+  StyledUserLink,
+  StyledDes,
+  StyledBuyRent,
+  StyledNav} from './styles'
+import { useParams } from "react-router-dom"
+import { useEffect } from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { getProduct } from '../../store/productReducer'
+
+function DetailedProduct() {
+    const { 
+      picture,
+      nameProduct,
+      brand,
+      quantity,
+      description,
+      buyPrice,
+      rentPrice,
+      user,
+    } = useSelector(({productReducer}) =>  productReducer.product )   
+    const dispatch = useDispatch()
+    
+    console.log(user)
```

Comment:
Ojo con los console logs

Reply:
Listo. Arreglado. Se me pasó ese haha

Reply:
@simonhoyos please, check the last changes done by Miguel, thanks in advance.